app/src/main/java/io/pumpkinz/pumpkinreader/service/ApiService.java

```diff
@@ -11,8 +11,20 @@
 
 public interface ApiService {
 
+    @GET("/newstories.json")
+    Observable<List<Integer>> listNew();
```

Comment:
I'm not fond of the method's naming here: ```listNew```
It's not very descriptive.
Better name would be ```getRecentNewsIds()```, ```getTopNewsIds()```, ```getHNJobIds```, ```getAskHNIds```, etc.